ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/radiobuttonsActivityPicklistQuestion.component.ts

```diff
@@ -8,7 +8,8 @@ import { NGXTranslateService } from '../../../services/internationalization/ngxT
     selector: 'ddp-activity-radiobuttons-picklist-question',
     template: `
     <div>
-        <mat-radio-group class="ddp-radio-group-column">
+        <label id="radio-group-label" *ngIf="block.label" [innerHTML]="block.label"></label>
```

Comment:
Is there anything we can do in `ActivityCompositeAnswer` component? From a cursory reading of the code, `shouldSetLabelToBeQuestionText()` returns `true` because we do have a "date picklist" child, which makes `convertQuestionToLabels` to be `true` and causing question prompt to be blanked out / converted to `label`?

Reply:
I think first we just need to decide how we want to present this question and work our solution back from that. I actually like that the we are producing a label element for the radio buttons, just don't like how it looks with current styling and in contrast with the sibling date question.
Does not help that in material angular, the radio buttons, radio groups are treated differently that the other input widgets regarding labels.
The general solutions in composite answer are starting to get complicated!
I am inclined for now to follow @fat23cat suggestion and merge and come back to this as we add more styling. to the whole system. He is still refining them.